Automattic/wp-calypso
# Add Sensei title and style onboarding screen

## Sensei Onboarding - pick style and site title

### Overview:
![senseisetup](https://user-images.githubusercontent.com/3220162/200428406-ab983390-f462-4130-8b11-4bfe2695bfc3.gif)

### Notes: 
- When the screen size get's smaller, the image becomes centered in the container as it gets smaller.  Another option would be to crop the image but I think centering it looks better
- The Site Title and the Style are both saved as `state` here - do we need to do anything further?  I'm assuming we can use the state later in the onboarding to set the `NewSiteBlogDetails`.   
- Right now they are just saved as colors like `green`.  I'm assuming we'll change this to match whatever option we need to set from the Sensei Theme when it's done.
- When you click on a different style, the image changes to show the style, but the Title does not change.  I felt like that would be maybe a big task for a small change so I opted to not do it.
- Right now, the `initialSiteTitle` is set to `test` - is that intended?  It looks like it's coming from the `ONBOARD_STORE`

#### Testing Instructions

- Start Calypso
- Go to http://calypso.localhost:3000/setup/senseiSetup?flow=sensei
- Change styles, change site title
- Click Continue and make sure you are able to select a domain and you see suggestions with your Site Title.

cc @dadish 

#### Pre-merge Checklist
